docs/docs/gatsby-cli.md

```diff
@@ -200,30 +202,30 @@ You can type in a command, such as one of these:
 
 `staticQueries`
 
-When combined with the [GraphQL explorer](/docs/introducing-graphiql/), these REPL commands could be very helpful for understanding your Gatsby site's data.
+[GraphQL explorer](/docs/introducing-graphiql/) के साथ उपयोग करने पर, ये आरईपीएल कमांड आपके Gatsby साइट के डेटा को समझने के लिए बहुत सहायक हो सकते हैं।
 
-For more information, check out the [Gatsby REPL documentation](/docs/gatsby-repl/).
+अधिक जानकारी के लिए, देखें [Gatsby REPL डॉक्युमेंटेशन](/docs/gatsby-repl/)
```

Comment:
`देखें [Gatsby REPL डॉक्युमेंटेशन](/docs/gatsby-repl/)` => `[Gatsby REPL डॉक्युमेंटेशन](/docs/gatsby-repl/) देखें।`

Reply:
`देखें।)` => `देखें।`

you have added an extra closing bracket `)`